src/gen_dragonfruit/gd_board.cpp

```diff
@@ -104,10 +104,9 @@ static void gd_board_post(){
 
     // Check MPL115a2 temperature
     int32_t mpl115a2_temp_val = gd_dev_adafruit_MPL115A2_temp_read();
-    Serial.print(F("[P] mpl115a2 value: "));
+    Serial.print(F("[P] mpl115a2 temp: "));
     Serial.print(mpl115a2_temp_val);
-    Serial.print(F("."));
-
+    Serial.println(" K");
```

Comment:
This statement should be enclosed in the F() macro declaration to save SRAM space.